var/spack/repos/builtin/packages/thepeg/package.py

```diff
@@ -63,6 +62,10 @@ class Thepeg(AutotoolsPackage):
 
     force_autoreconf = True
 
+    depends_on("c", type="build")
+    depends_on("cxx", type="build")
+    depends_on("fortran", type="build")
```

Comment:
The error in https://gitlab.spack.io/spack/spack/-/jobs/14997553 doesn't refer to fortran. Can't find any references to fortran in thepeg-2.3.0.  (C is pulled in by AX_CHECK_ZLIB)
```suggestion
```